frontend/src/Components/NavigationWrapper/NavigationListItem.tsx

```diff
@@ -0,0 +1,33 @@
+import { ListItem, ListItemIcon, ListItemText } from "@material-ui/core";
+import React, { ReactNode } from "react";
+import { RouteComponentProps, withRouter } from "react-router-dom";
+
+interface IProps extends RouteComponentProps<any> {
+  text: string;
+  children: any;
```

Comment:
_`any`_  ? _any_ ? _ANY_ ? 
don't do it.
Gali gal pabandyt:
```typescript
const NavigationListItem: FunctionComponent<IProps> = ({ [..], children }) => {[..]}
```

https://fettblog.eu/typescript-react/children/

Reply:
Pabandysiu

Reply:
Visaip bandziau, niekaip neveikia, jei paciam iseis padaryt tai labai welcome :^)

Reply:
veikia su `React.ReactElement`, tada žinoma galima tik vieną elementą perduot, bet visada galima apwrappint su `Fragment`